mmcv/ops/nms.py

```diff
@@ -248,14 +247,14 @@ def batched_nms(boxes, scores, idxs, nms_cfg, class_agnostic=False):
         boxes_for_nms = boxes
     else:
         max_coordinate = boxes.max()
-        offsets = idxs.to(boxes) * (max_coordinate + 1)
+        offsets = idxs.to(boxes) * (max_coordinate + torch.tensor(1).to(boxes))
```

Comment:
We may leave a comment here.

Reply:
Basically, it converts python scalar int variable to tensor for tracing. In case of converting models to onnx using cuda, it may be more friendly. It's also the same as what torchvision did in [here](https://github.com/pytorch/vision/blob/b2171653f83543d380715ddcffb5ebb2d05efa2e/torchvision/ops/boxes.py#L86)

Reply:
Please leave the comment in the code thus other developers will be notified when doing the related modifications.